backend/datahubhel/settings.py

```diff
@@ -20,7 +20,8 @@
     VAR_ROOT=(str, default_var_root),
     LOG_ROOT=(str, default_log_root),
     ALLOWED_HOSTS=(list, []),
-    DATABASE_URL=(str, 'postgis:///datahubhel'),
+    DATABASE_URL=(str,
+                  'postgis://datahubhel:datahubhel@datahubhel_db/datahubhel'),
```

Comment:
Don't change the default here, but rather set the environment variable in the docker-compose.yml.